test/pylib/scylla_cluster.py

```diff
@@ -707,9 +707,21 @@ async def stop(self) -> None:
             # this step to fail unconditionally.
             pass
         await self.shutdown_control_connection()
+
+        if self.cmd.returncode is not None:
+            # process has already exited
+            if self.cmd.returncode != 0:
+                self.logger.error("%s exited with non-zero status code: %d", self, self.cmd.returncode)
```

Comment:
For example:
Grepping the logs for `exited` becomes consistent - you find something for sure - either a success or a fail.
Omitting the success is almost the same except that the success path is implicit by finding nothing.

Reply:
i don't grep log for "exited". probably we can wait for more inputs from those who check the log messages on daily basis. IMHO, the "consistency" is not the only thing we care about.

Reply:
It is not consistency it is usability.
Reporting one way _and_ the other is much more user friendly than reporting just the error case.
Imagine someone seeking for problems - knowing positively that the exit was done correctly is much better than the need to know that no report is a good report.

Reply:
so, let's wait for others' input.

and, we add code because it helps and it addresses our pain. not because it would not disturb. for instance, we don't copy the works of Shakespeare as a comment in this script just because it would not disturb.

Reply:
I rather see positive exit as well, and then if we figure out it just noise - remove it later.